feat(changelog): instances-added-debian-"bookworm"-is-now 2023-06-13 (#1799)
scaleway/docs-content: changelog/june2023/2023-06-13-instances-added-debian-bookworm-is-now.mdx

---

> title: Debian Bookworm is now available for your instances

Reviewer: ```suggestion
title: Debian Bookworm is now available for your Instances
```

> title: Debian Bookworm is now available for your instances

Reviewer 2: ```suggestion
title: Debian Bookworm is now available for your Instances
```

status: added
author:

>     fullname: 'Join the #[instance] channel on Slack.'

Reviewer 2: ```suggestion
    fullname: 'Join the #instance channel on Slack.'
```

>     fullname: 'Join the #[instance] channel on Slack.'

Reviewer: ```suggestion
    fullname: 'Join the #instance channel on Slack.'
```

    url: 'https://slack.scaleway.com'
date: 2023-06-13
category: compute
product: instances
---

> New addition to our list of supported OS image for your instances : Debian 12 Bookworm, just released on June 10th, 2023 is available immediately !

Reviewer 3: ```suggestion
A new addition to our list of supported OS images for your instances: Debian 12 Bookworm, released on June 10th, 2023 is available now!
```
